src/services/document_service.py

```diff
@@ -173,9 +178,10 @@ def get_node_by_uid(self, data_source_id: str, document_uid: str, depth: int = 9
         complete_document = get_complete_sys_document(
             document_uid, self.repository_provider(data_source_id, self.user), depth
         )
-        return Node.from_dict(
+        return tree_node_from_dict(
             complete_document,
             complete_document.get("_id"),
+            key="",
```

Comment:
looks like key is set to "" in the majority of the tree_node_from_dict() function calls.

Maybe better to make key have a default value None or "" in tree_node_from_dict() and override it necessary?

Reply:
fixed!
Did the same for UID